README.md

```diff
@@ -8,7 +8,7 @@ It tunes some options from upstream to make the chart easier to deploy.
 
 This chart is meant to be used with S3 compatible storage only. Access to the S3
 storage must be ensured for the chart to work. You can check
-[the sample config file](sample_configs/values-gs.yaml) to check for annotations
+[the sample config file](https://github.com/giantswarm/loki-app/blob/master/sample_configs/values-gs.yaml) to check for annotations
 that can be used to make it work with AWS S3 using
 [KIAM](https://github.com/uswitch/kiam). Check [below](#deploying-on-aws) to see
 what configuration you need on the AWS side.
```

Comment:
This section is very AWS specific and do not belong here.

Reply:
Fixed.